docs/content-services/components/search-date-range-advanced-tabbed.component.md

```diff
@@ -0,0 +1,103 @@
+---
+Title: Search date range advanced component
```

Comment:
Should not be "Search date range advanced tabbed component"?

Reply:
Updated the title. This was probably a copy paste miss. Thanks for pointing it